code/modules/plumbing/plumbers/bottler.dm

```diff
@@ -52,15 +56,28 @@
 			inputspot = get_step(get_turf(src), WEST)
 			badspot = get_step(get_turf(src), SOUTH)
 
+	//If by some miracle
+	if( ( !valid_output_configuration ) && ( goodspot != null && inputspot != null && badspot != null ) )
+		valid_output_configuration = TRUE
+		begin_processing()
+
 ///changing input ammount with a window
 /obj/machinery/plumbing/bottler/interact(mob/user)
 	. = ..()
+	if(!valid_output_configuration)
+		to_chat(user, span_warning("Output location error condition warning is flashing on the display"))
+		return .
```

Comment:
```suggestion
		return
```

Reply:
better to be explicit than implicit

Reply:
I hate `.` and always do stuff like this